cluster-logging.md

```diff
@@ -0,0 +1,114 @@
+| `k8s.namespace.name` | resource | container | stream label | |
+| `k8s.container.name` | resource | container | stream label | |
+| `k8s.pod.labels.*` | resource | container | structured metadata | |
+| `k8s.pod.name` | resource | container | stream label | |
+| `k8s.pod.uid` | resource | container | stream label | |
+| `k8s.cronjob.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
+| `k8s.daemonset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
+| `k8s.deployment.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
+| `k8s.job.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
+| `k8s.replicaset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
+| `k8s.statefulset.name` | resource | container | structured metadata | Conditionally forwarded based on creator of Pod |
+| `log.iostream` | log | container | structured metadata | |
+| `k8s.audit.event.level` | log | audit | structured metadata | |
+| `k8s.audit.event.stage` | log | audit | structured metadata | |
+| `k8s.audit.event.user_agent` | log | audit | structured metadata | |
+| `k8s.audit.event.request.uri` | log | audit | structured metadata | |
+| `k8s.audit.event.response.code` | log | audit | structured metadata | |
+| `k8s.audit.event.annotation.*` | log | audit | structured metadata | |
+| `k8s.audit.event.object_ref.resource` | log | audit | structured metadata | |
+| `k8s.audit.event.object_ref.name` | log | audit | structured metadata | |
+| `k8s.audit.event.object_ref.namespace` | log | audit | structured metadata | |
+| `k8s.audit.event.object_ref.api_group` | log | audit | structured metadata | |
+| `k8s.audit.event.object_ref.api_version` | log | audit | structured metadata | |
+| `k8s.user.username` | log | audit | structured metadata | |
+| `k8s.user.groups` | log | audit | structured metadata | |
+| `process.executable.name` | resource | journal | structured metadata | |
+| `process.executable.path` | resource | journal | structured metadata | |
+| `process.command_line` | resource | journal | structured metadata | |
+| `process.pid` | resource | journal | structured metadata | |
+| `service.name` | resource | journal | structured metadata | |
+| `systemd.t.*` | log | journal | structured metadata | |
+| `systemd.u.*` | log | journal | structured metadata | |
+
+**Note:** Attributes marked as "Compatibility attribute" are added to support minimal backwards compatibility with the [ViaQ](https://github.com/openshift/cluster-logging-operator/blob/release-6.0/docs/reference/datamodels/viaq/v1.adoc) data model. These attributes should be considered deprecated and will be removed one release after **General Acceptance** of Red Hat OpenShift Logging.
+
```

Comment:
Do we want to add a note on how the attributes will look on the Loki side?

```suggestion

**Note:** Loki changes the attribute names when persisting them to storage. They will be lower-cased and all dots (`.`) will be replaced by underscores (`_`). For example, `k8s.namespace.name` will become `k8s_namespace_name`.
```

Reply:
I will add the note and further clarify replacement of forward slash which additionally addresses labels

Reply:
@xperimental is :arrow_up: a correct statement